app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
That's a great Sandi Metz article. I just read it again, and I am confident that we have the right abstraction for non-Devise attributes. Email is an exception, so I think it should probably be left alone as opposed to force-fitted into this abstraction, as Peter has done in #1070.